### PR TITLE
Closes SiCKRAGETV/sickrage-issues/issues/2461

### PR DESCRIPTION
**Pull request by miigotu**

### CODE REVIEW

File: sickbeard/common.py [hunk 1]
```diff
@@ -40,6 +40,8 @@
               ]
 
 INSTANCE_ID = str(uuid.uuid1())
+if len(sickbeard.CUR_COMMIT_HASH) == 40:
+    INSTANCE_ID += '-' + sickbeard.CUR_COMMIT_HASH[:7] # Append github style short hash
```
Comment (WebSpider) [inline sickbeard/common.py]: sickbeard config variables aren't available here yet, if i read the code correctly?

**Comment by miigotu** [comment]
@WebSpider yes I saw that, and I cant even get them to there either....

**Comment by WebSpider** [comment]
Yeah i was having similar troubles with causing circular imports..

**Comment by miigotu** [comment]
I think this might be because of sickbeard/__init__.py importing like `from sickbeard import` where is should be `from . import` or just `import `

**Comment by miigotu** [comment]
eh, because of `__init__` importing from common. maybe need to mod the USER_AGENT later, this one is a pita.

**Pull request closed without merging**